pkgs/development/web/playwright/driver.nix

```
@@ -161,13 +161,13 @@ let
       browsersJSON = (lib.importJSON ./browsers.json).browsers;
       browsers =
         {
-          x86_64-linux = browsers-linux { };
-          aarch64-linux = browsers-linux { };
-          x86_64-darwin = browsers-mac;
-          aarch64-darwin = browsers-mac;
+          x86_64-linux = browsers { };
+          aarch64-linux = browsers { };
+          x86_64-darwin = browsers { };
+          aarch64-darwin = browsers { };
         }
         .${system} or throwSystem;
-      browsers-chromium = browsers-linux { };
+      browsers-chromium = browsers { };
```

should we change this?
```suggestion
      browsers-chromium = browsers { 
        withWebkit = false;
        withFirefox = false;
      };
```